src/Components/fe-login/test/fe-login.test.js

```diff
@@ -21,6 +32,27 @@ describe('FeLogin', () => {
       username: 'username',
       password: 'password',
     });
+    requestMock.restore();
+    expect(username.modelValue).to.equal('');
+    expect(password.modelValue).to.equal('');
+    expect(notification.classList.contains('hidden')).to.be.true;
+  });
+
+  it('checks for the error in the server', async () => {
+    const username = element.shadowRoot.querySelector('#username');
+    const password = element.shadowRoot.querySelector('#password');
+    const notification = element.shadowRoot.querySelector('fe-notification');
+    const requestMock = sinon.stub(ajax, 'requestJson');
+    requestMock.rejects({});
+    username.modelValue = 'username';
+    password.modelValue = 'password';
+
+    element.login();
+    await aTimeout(10);
+    requestMock.restore();
+    expect(notification.classList.contains('hidden')).to.be.false;
+    expect(username.modelValue).to.equal('username');
```

Comment:
what is the use of this expect ?, you only added the modelValue and you itself checking that same as equal ?

Reply:
ok will change